packages/office-ui-fabric-react/src/components/Modal/Modal.base.tsx

```diff
@@ -167,19 +234,20 @@ export class ModalBase extends BaseComponent<IModalProps, IDialogState> implemen
           >
             <div className={classNames.root}>
               {!isModeless && <Overlay isDarkThemed={isDarkOverlay} onClick={isBlocking ? undefined : (onDismiss as any)} />}
-              <FocusTrapZone
-                componentRef={this._focusTrapZone}
-                className={classNames.main}
-                elementToFocusOnDismiss={elementToFocusOnDismiss}
-                isClickableOutsideFocusTrap={isModeless || isClickableOutsideFocusTrap || !isBlocking}
-                ignoreExternalFocusing={ignoreExternalFocusing}
-                forceFocusInsideTrap={isModeless ? !isModeless : forceFocusInsideTrap}
-                firstFocusableSelector={firstFocusableSelector}
-              >
-                <div ref={this._allowScrollOnModal} className={classNames.scrollableContent} data-is-scrollable={true}>
-                  {this.props.children}
-                </div>
-              </FocusTrapZone>
+              {isDraggable ? (
+                <DraggableZone
```

Comment:
FWIW I see the `DraggableZone` adding about 10k minified to the bundle. It's not great but it is a lot better.